.github/workflows/ci.yaml

```diff
@@ -0,0 +1,98 @@
+name: CI
+
+on:
+  workflow_dispatch:
```

Comment:
Is this for manually running the workflow?

Reply:
yes.